packages/esm-ohri-pmtct-app/src/views/mch-summary/tabs/current-pregnancy.component.tsx

```diff
@@ -237,58 +252,29 @@ const CurrentPregnancy: React.FC<PatientChartProps> = ({ patientUuid }) => {
       {
         key: 'motherHIVStatus',
         header: t('motherHIVStatus', 'Mother HIV Status'),
-        encounterTypes: [encounterTypes.motherPostnatal, encounterTypes.labourAndDelivery, encounterTypes.antenatal],
-        getObsValue: (encounters) => {
-          const pncArtData = {
-            artInitiation: getObsFromEncounter(encounters[0], obsConcepts.artInitiationConcept),
-            motherHIVStatus: getObsFromEncounter(encounters[0], obsConcepts.hivTestResultConcept),
-            pTrackerId: getObsFromEncounter(encounters[0], obsConcepts.pTrackerIdConcept),
-          };
-          const lndArtData = {
-            artInitiation: getObsFromEncounter(encounters[1], obsConcepts.artInitiationConcept),
-            motherHIVStatus: getObsFromEncounter(encounters[1], obsConcepts.hivTestResultConcept),
-            pTrackerId: getObsFromEncounter(encounters[1], obsConcepts.pTrackerIdConcept),
-          };
-          const ancArtData = {
-            artInitiation: getObsFromEncounter(encounters[2], obsConcepts.artInitiationConcept),
-            motherHIVStatus: getObsFromEncounter(encounters[2], obsConcepts.hivTestResultConcept),
-            pTrackerId: getObsFromEncounter(encounters[2], obsConcepts.pTrackerIdConcept),
-          };
-          const latestArtData = getLatestArtDetails(pncArtData, lndArtData, ancArtData);
-          if (!latestArtData['motherHIVStatus']) {
-            return '--';
-          }
-
-          return latestArtData['motherHIVStatus'];
+        encounterTypes: [encounterTypes.labourAndDelivery],
+        getObsValue: async ([encounter]) => {
```

Comment:
Are we using the encounter in anyway in these instances if the data if already fetched from a network request? If not I think we should clean this up for readability.